code/game/objects/random/loot.dm

```diff
@@ -356,7 +356,10 @@
 		/obj/item/clothing/mask/gas/voice = 1,
 		/obj/item/clothing/gloves/brassknuckles = 2,
 		/obj/item/reagent_containers/inhaler/xuxigas = 1,
-		/obj/item/toy/balloon/syndicate = 4
+		/obj/item/toy/balloon/syndicate = 4,
+		/obj/item/material/knife/butterfly/switchblade/pocket_knife = 2.5,
+		/obj/item/material/knife/butterfly/switchblade/pocket_knife/blue = 2.5,
+		/obj/item/material/knife/butterfly/switchblade/pocket_knife/red = 2.5
```

Comment:
Spawn chance should be much lower and should be shared with normal butterfly knives.

Not only this has a *higher* chance of rolling, it's also rolling *three times*